source/simulator/lateral_averaging.cc

```diff
@@ -736,6 +793,13 @@ namespace aspect
                                 &this->get_gravity_model(),
                                 this->convert_output_to_years()));
           }
+        else if (property_names[property_index] == "rising_velocity")
+          {
+            functors.push_back(std_cxx14::make_unique<FunctorDepthAverageRisingVelocity<dim>>
```

Comment:
```suggestion
            functors.push_back(std::make_unique<FunctorDepthAverageRisingVelocity<dim>>
```

Reply:
I updated it in the other places too, just to be consistent.